Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -1154,8 +1219,8 @@ protected class ClassWriter : IClassWriter {
       return block;
     }
 
-    static string Keywords(bool isPublic = false, bool isStatic = false, bool isExtern = false) {
-      return (isPublic ? "public " : "") + (isStatic ? "static " : "") + (isExtern ? "extern " : "");
+    static string Keywords(bool isPublic = false, bool isStatic = false, bool isExtern = false, bool isVirtual = true) {
```

Comment:
Where is this new parameter used?

Reply:
I only ever used the default value, so I removed the parameter.